sdcm/db_stats.py

```diff
@@ -326,7 +326,15 @@ class TestStatsMixin(Stats):
     """
     This mixin is responsible for saving test details and statistics in database.
     """
-    KEYS = ['test_details', 'setup_details', 'versions', 'results', 'nemesis', 'errors', 'coredumps']
+    KEYS = {
+        'test_details': {},
+        'setup_details': {},
+        'versions': {},
+        'results': {},
+        'nemesis': {},
+        'errors': [],
+        'coredumps': {}
+    }
```

Comment:
maybe for clarity we should do that ?
```suggestion
    KEYS = {
        'test_details': dict(),
        'setup_details': dict(),
        'versions': dict(),
        'results': dict(),
        'nemesis': dict(),
        'errors': list(),
        'coredumps': dict()
    }
```

Reply:
Thanks Israel, fixed.